test/Sentry.Extensions.Logging.EfCore.Tests/Sentry.Extensions.Logging.EfCore.Tests.csproj

```diff
@@ -0,0 +1,37 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp2.1;net461</TargetFrameworks>
+  </PropertyGroup>
+  
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1' OR '$(TargetFramework)' == 'net461'">
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.17" />
```

Comment:
Confusing version setup, 3.1 came out with ,NET Core 3.1, why require it here instead of 2.1?

Worth noting: netcoreapp2.1 is EOL next month. EF Core on .NET Framework 4.8 is likely not a target.

Reply:
I chose Version 3.1.17 because it's the latest version compatible with .NET 461 and .NET Core 2.1 (requires .NET Standard 2.0) while the latest version requires .NET Standard 2.1

In short, I chose  the same TargetFrameworks that are used by Extensions Logging Tests, if you prefer I could limit it to just net5 and netcore 3.1 .